OvieMudi/Banka
# #165391832 Add a simple responsive landing page

#### What does this PR do?
- Adds a landing page to the project
- Use visible CTA links to signup and pages
#### Description of Task to be completed?
- Go to the URL at https://oviemudi.github.io/Banka/UI/
- Ensure the page is responsive
#### How should this be manually tested?
- Start the local server
- Go to local page at `localhost:<PORT>/UI/index.html `
- Ensure the page is working as desired
#### Any background context you want to provide?
This feature ensures that a user can perform actions such as sign up and sign in easily.
#### What are the relevant pivotal tracker stories?
[#165391832 ](https://www.pivotaltracker.com/story/show/165391832 )
#### Screenshots (if appropriate)
![homepage1](https://user-images.githubusercontent.com/42499892/56267966-320e7c00-60e8-11e9-8c8a-301d2a324949.PNG)
![hompage2](https://user-images.githubusercontent.com/42499892/56267992-3fc40180-60e8-11e9-8758-a32e6b52eefa.PNG)
![hompage3](https://user-images.githubusercontent.com/42499892/56268006-4ce0f080-60e8-11e9-938e-ad0f2c47d67e.PNG)
![hompage4](https://user-images.githubusercontent.com/42499892/56268021-5a967600-60e8-11e9-98ba-78bd3da83a56.PNG)
